pandas/_typing.py

```diff
@@ -82,6 +83,7 @@
 
 Axis = Union[str, int]
 Label = Optional[Hashable]
+IndexLabel = Optional[Union[bool, str, Sequence[Label]]]
```

Comment:
shouldn't this just be `Optional[Union[Label, Sequence[Label]]]` ?

Reply:
cc @simonjayhawkins

Reply:
As far as I understand, index label can be bool (at least in the original implementation). So, I left it exactly as it was.
It can probably be changed into ``Optional[Union[bool, Label, Sequence[Label]]]``, if needed.

Reply:
*bool* should be included in Hashable though? if that's the case then I don't think you need it here (or is somthing breaking)?

Reply:
Replaced as IndexLabel suggested in b7dae112